src/utils/__tests__/idpLoader.test.ts

```diff
@@ -420,7 +420,8 @@ describe("spidStrategy#loadFromRemote", () => {
   it("load idp options with missing idps configurations", async () => {
     const loadFromRemote = require("../../strategies/spidStrategy")
       .loadFromRemote;
-    const idpOptions = await loadFromRemote(IDPMetadataUrl);
+    const idps = require("../../strategies/spidStrategy").IDP_IDS;
+    const idpOptions = await loadFromRemote(IDPMetadataUrl, idps);
```

Comment:
I'd add a test where `loadFromRemote` throws an error (if there isn't one yet)

Reply:
The same of [here](https://github.com/teamdigitale/io-backend/pull/558#discussion_r365874603)